Makefile

```diff
@@ -2,7 +2,7 @@ SHELL := /bin/bash
 
 # Current Operator version
 VERSION ?= latest
-CSV_VERSION = $(VERSION)
+CSV_VERSION = $(shell echo $(VERSION) | sed 's/v//')
```

Comment:
This really isn't important, but I would tend to use `:=` unless you know you need the variable evaluated each time its used.  The behavior of `:=`, which is to evaluate where it's defined, is a bit more clear (and maybe ever so slightly faster if it's a variable used a bunch of times).